### PR TITLE
Fix testImageLabels

### PR DESCRIPTION
**Pull request by davidxia**
Account for pre-existing images with the same IDs.
This happens when running the test multiple times.
See comment in the code.

### CODE REVIEW

**Comment by davidxia** [comment]
@mattnworb @johnflavin

File: src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java [hunk 1]
```diff
@@ -2220,6 +2220,14 @@ public void testImageLabels() throws DockerException,
     final List<Image> preexistingTestImages = sut.listImages(
             ListImagesParam.withLabel("name", "testtesttest"));
     final int numPreexistingTestImages = preexistingTestImages.size();
+    // Make a list of just the image IDs
+    final List<String> preexistingTestImageIds =
+        Lists.transform(preexistingTestImages, new Function<Image, String>() {
```
Comment (johnflavin) [inline src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java]: Could just do

    final List<String> preexistingTestImageIds = imagesToShortIds(preexistingTestImages);

if you’re ok with the short (12-hex long) id.

Comment (davidxia) [inline src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java]: Thanks. I forgot about that function.

**Comment by johnflavin** [comment]
Thanks for fixing this. I wasn't aware it was a problem.

Ideally none of this `preExistingImages` stuff would be necessary. If we could build an image with attributes from a POST body instead of from a Dockerfile, it would be easy to give the test images some random label value at runtime, and then you could be (almost) certain to get no collisions on the labels. But it is what it is. `¯\_(ツ)_/¯`

**Comment by mattnworb** [comment]
:+1:

**Comment by davidxia** [comment]
The tests were still failing so I just removed that part of the test. see https://github.com/spotify/docker-client/commit/fe9083a20249cb010b6f71e8968b3888145a0592

**Pull request merged**